Manifest.toml

```diff
@@ -241,6 +241,29 @@ git-tree-sha1 = "71c79e77221ab3a29918aaf6db4f217b89138608"
 uuid = "b305315f-e792-5b7a-8f41-49f472929428"
 version = "1.0.1"
 
+[[deps.Enzyme]]
+deps = ["CEnum", "EnzymeCore", "Enzyme_jll", "GPUCompiler", "LLVM", "Libdl", "LinearAlgebra", "ObjectFile", "Preferences", "Printf", "Random"]
+git-tree-sha1 = "ae881b2f107e3c01444edbaa0bf3b73f461539f6"
+repo-rev = "main"
```

Comment:
it's better to depend on a tagged version of a package

Reply:
Hmm yeah, let's change that to the latest